MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+The Mina Foundation will create a pull request to create a new page to host the MIP template inside the MIPs section in GitHub.
+https://docs.google.com/document/d/1WUORti_GyXikuGPIf9Xxr7otauDzidDhk__ElDVAfC0/edit
+
+**Add new communication channels in Discord for MIP Facilitators and MIP Reviewers:**
+The Admins of the Mina Discord Server will create new private channels to coordinate the operations and internal discussion for both of these roles, granting MIP Facilitators the necessary permissions to effectively facilitate communications and moderate these channels as needed. 
+
+**Implementation plan**
+The Mina Foundation, in collaboration with o1 Labs and other members of the Mina ecosystem, will support the implementation of the proposed changes and help initiate this new process by temporarily filling the new roles as needed while the processes are tested, and the community develops the capacity to effectively assume these roles.
+
+These changes should be implemented, and the new roles filled by elected community members, no later than one year after approval through an on-chain vote. The Mina Foundation will provide regular updates and coordinate communications throughout the implementation phase.
+
+
+## Security Considerations
+
+**Dangerous proposals passing with low turnout**
+The proposal introduces Risk Assessments for informational purposes, allowing the community to review risks before voting. However, the proposal depends on the community to stay informed and reject dangerous proposals through on-chain voting. To further mitigate this, future upgrades could consider linking risk levels to higher participation thresholds for high-risk MIPs.
+
+**Concentrating too much power in the hands of a few experts**
+The risk of centralizing authority in the hands of a few reviewers is mitigated through transparency and broader community involvement. Public debates, deliberation tools, and well-defined reviewer roles ensure decisions reflect diverse perspectives and are not overly influenced by a small group.
+
+**Risk of not having enough experts**
+The proposal addresses the potential lack of expertise by assigning MIP Facilitators to manage the process, ensuring that a sufficient number of expert reviews are collected. If expertise is lacking, the MIP may be put on hold until necessary input is secured.
+
+**Conflict of Interest between MIP Author and Reviewers**
+To prevent conflicts of interest, the proposal establishes separate roles for authors and reviewers. MIP Facilitators ensure that authors do not review their own proposals, promoting objectivity and impartiality in the evaluation process.
+
+**Managing capacity for MIP reviews**
+MIP Facilitators are responsible for managing MIP flow, estimating future volumes, and requesting additional resources as needed. They develop strategies to handle capacity overload, ensuring the process remains efficient during periods of high activity.
```

Comment:
What sort of resources are they expected to have/request other than sufficient Facilitators and Reviewers? This may be confusing.

Reply:
My initial thought was indeed that additional facilitators and reviewers could be needed during periods of high activity. However, it's also possible that other resources, like tools or software for managing MIP flow or tracking progress, might also be necessary as the process becomes more complex. At this point, it's difficult to fully predict what additional resources will be required. The intention of this section is to offer flexibility, as the exact needs may evolve as we test and refine the process.